enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -782,8 +760,13 @@ flowchart LR
 The Infrastructure Operator is generally responsible for installing and configuring the Gateway API provider
 (e.g., Istio), installing the Gateway API CRDs, and creating the GatewayClass. However, since the Ingress Operator
 handles the installation of OSSM and the CRDs, the Infrastructure Operator is only responsible for creating
-and managing the GatewayClass. Cluster Operators create and manage the Gateways, while Application Developers create
-and manage the routes.
+and managing the GatewayClass. Cluster Operators create and manage the Gateways and ReferenceGrants, while
+Application Developers create and manage the xRoutes.
+
+> **Note**: While initially ReferenceGrant will be limited to cluster-admin
+> scope due to its security sensitive nature, we will consider and re-evaluate
+> the need for more specific scopes for this resource given user feedback over
+> time.
```

Comment:
```suggestion
> time.  In the meantime, application developers that provide access across namespaces will need to ask the cluster admin to provision those permissions.
```

Reply:
Slight rewording, because I worry it sounds like we're encouraging all namespace, but I think we should be saying the "default" position is single namespace:

```suggestion
> time.  We expect the cluster-admin will need to allow all namespaces in some cases.  In the meantime, application developers that limit access to only certain namespaces will need to ask the cluster admin to provision those permissions or add a role to allow application developers to allow a subset of namespaces to access their application. 
```

Let me know if that's :+1: or if you'd still like to see further changes.